plugins/scaffolder/src/components/fields/EntityPicker/EntityPicker.tsx

```diff
@@ -38,6 +41,37 @@ export interface EntityPickerUiOptions {
   defaultNamespace?: string | false;
 }
 
+export const EntityPickerSchema: FieldExtensionSchema = {
+  input: {
```

Comment:
Wonder if it's possible to re-use these schemas to generate the `typescript` types for the `FieldExtension` components. Thought of doing this before, and I wonder if we can lean on a library to do this for us. It's a little unfortunate that these types are duplicated.

Alternatively we've thought about support `zod` or something instead to define the input and output schema which supports typescript and then https://www.npmjs.com/package/zod-to-json-schema to generate the json schema for input and output.